Makefile

```diff
@@ -0,0 +1,12 @@
+# HELP
```

Comment:
I wouldn't add this file, considering is not cross-platform and doesn't serve a bigger purpose other than running some standard commands to generate docs. Additionally, it's not related with the PR goal ;)

Reply:
make sense